doc/taxonomy.md

```diff
@@ -143,3 +143,137 @@ The protocol has a fixed list of values defined at the data model (as shown belo
 |Proxy|Contract that acts as a proxy to another contract and is used to upgrade the logic of the target contract without changing its address|
 |Exchange|Addresses used by centralized or decentralized exchanges to interact with smart contracts and blockchain networks|
 
+## Schema
+
+### Label Level Schema
+
+```json
+{
+  "address": "string",
+  "name": "string",
+  "source": {
+    "type": "string",
+    "enum": ["Ground Truth", "Heuristic", "Machine Learning", "Research", "External"]
```

Comment:
1) https://github.com/microscopexyz/chainmetareader/blob/main/doc/taxonomy.md (contains"research") vs. 2) https://docs.microscopeprotocol.xyz/onboarding/data/source (doest not contain "research"). this difference, I do not know where or when we introduced. Anyway, it should open an other discussion thread to make them converge. Regardlessly, it should not impact this PR

Reply:
@tamamatammy @zhanwu within Coinbase, is there significant portion of labels from "research"? To simplify the array of source-type, is it okay to this source into "ground truth" (if the research can draw solid conclusion) or "heuristics" (it utilizes the investigation based on supporting material to draw a likely conclusion)?

Reply:
@beingzy we can update the array if needed 

@tamamatammy what should be in the array above? We certainly want to constrain the taxonomy